client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidEditTextTest.java

```diff
@@ -31,8 +31,8 @@
     
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String textFieldLocator = "com.paypal.selion.pageobjectsdemoapp:id/edit_text";
+    private final String actionButtonLocator = "com.paypal.selion.pageobjectsdemoapp:id/action_button";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120313/issues/source?bid=2928425&fileBranchId=2943483#l34)